assembl/static/js/app/common/collectionManager.js

```diff
@@ -344,7 +344,7 @@ var CollectionManager = Marionette.Object.extend({
    * @returns {Promise}
    * @function app.common.collectionManager.CollectionManager.getMessageFullModelRequestWorker
   */
-  getMessageFullModelRequestWorker: function(collectionManager) {
+  getMessageFullModelRequestWorker: function(collectionManager, messagesStructureCollectionPromise) {
```

Comment:
@swergas Can you comment as to why this was now injected into the request worker? I knew this, and even I forgot :p